Quaver.Shared/Localization/LocalizedString.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Quaver.Shared.Localization
+{
+    /// <summary>
+    ///     Parameter to <see cref="LocalizationManager.Get(LocalizedString, object[])"/>
+    /// </summary>
+    public enum LocalizedString
+    {
+        FPS_IS_NOW_UNLIMITED, FPS_IS_NOW_LIMITED_TO_240_FPS, VSYNC_ENABLED, FPS_IS_NOW_CUSTOM_LIMITED_TO, HOME
```

Comment:
This should be removed, but it's worth mentioning that each one should be on new lines and also not fully capitalized.